### PR TITLE
Add mariadb 10.6 support

### PR DESCRIPTION
**Pull request by samanthajayasinghe**

### CODE REVIEW

File: compose/mariadb.yml [hunk 1]
```diff
@@ -2,15 +2,31 @@ version: "3.7"
 services:
 
   mariadb:
```
Comment (derschatta) [inline compose/mariadb.yml:4]: changing the default here does create a few issues if you are already using this container, people would have to delete the old container and volumes before being able to use it.

I wonder if it's better to leave that one for now or just don't have a generic mariadb container but name all db containers with their specific version

File: compose/mariadb.yml [hunk 2]
```diff
@@ -2,15 +2,31 @@ version: "3.7"
...
+      - nginx
+    networks:
+      - totara
+
+  mariadb105:
     image: mariadb:10.5
     container_name: totara_mariadb105
     ports:
-      - "3307:3306"
+      - "3310:3306"
```
Comment (derschatta) [inline compose/mariadb.yml:24]: we should not change the port as developers have set it up already

File: compose/mariadb.yml (hunk 1 above)
Comment (derschatta) [reply compose/mariadb.yml:4]: when we release this we have to put some notes in the release notes

**Review by derschatta — CHANGES_REQUESTED** [review]

**Comment by markmetcalfe** [comment]
Comments resolved in #163

**Pull request closed without merging**